src/hooks/useIndicatorStatus.ts

```diff
@@ -0,0 +1,74 @@
+import {useOnyx} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import {isConnectionInProgress} from '@libs/actions/connections';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import * as SubscriptionUtils from '@libs/SubscriptionUtils';
+import * as UserUtils from '@libs/UserUtils';
+import * as PaymentMethods from '@userActions/PaymentMethods';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import useTheme from './useTheme';
+
+type IndicatorStatus = {
+    indicatorColor: string;
+    status: ValueOf<typeof CONST.INDICATOR_STATUS> | undefined;
+    idOfPolicyWithErrors: string | undefined;
+};
+
+function useIndicatorStatus(): IndicatorStatus {
```

Comment:
```suggestion
type IndicatorStatus = ValueOf<typeof CONST.INDICATOR_STATUS>;

type IndicatorStatusResult = {
    indicatorColor: string;
    status: IndicatorStatus | undefined;
    policyIDWithErrors: string | undefined;
};

function useIndicatorStatus(): IndicatorStatusResult {
```

So you can reuse `IndicatorStatus` below and in other files